src/resolvers.ts

```diff
@@ -1,163 +1,109 @@
 import { GraphQLError } from 'graphql'
-import * as facilityService from './services/facilityService'
-import * as healthcareProfessionalService from './services/healthcareProfessionalService'
-import * as gqlType from './typeDefs/gqlTypes'
-import * as submissionService from './services/submissionService'
-import { CustomErrors, Result } from './result'
+import * as facilityService from './services/facilityService.js'
+import * as healthcareProfessionalService from './services/healthcareProfessionalService.js'
+import * as gqlType from './typeDefs/gqlTypes.js'
+import * as submissionService from './services/submissionService.js'
+import { Result } from './result.js'
```

Comment:
Why are these `.ts` files being imported as `.js`? 🤔

Reply:
Tl;dr; cause the latest compilers don't yet support it as .ts 🤷‍♂️ it's the best approach to this date.